mlflow/pyfunc/model.py

```diff
@@ -471,3 +501,9 @@ def predict(self, model_input, params: Optional[Dict[str, Any]] = None):
             )
         _log_warning_if_params_not_in_predict_signature(_logger, params)
         return self.python_model.predict(self.context, self._convert_input(model_input))
+
+
+def _get_pyfunc_loader_module(python_model):
+    if isinstance(python_model, ChatModel):
+        return mlflow.pyfunc.loaders.chat_model.__name__
+    return __name__
```

Comment:
what do we think of adding new pyfunc loaders to the `mlflow.pyfunc.loaders` module? i think it would be a clean way for us to implement future custom loaders (e.g. for `RAGModel`, `CompletionModel`).

Reply:
Sounds good to me.